src/main/java/entities/Ship.java

```diff
@@ -1,45 +1,47 @@
 package entities;
 
-import java.util.ArrayList;
 import java.util.HashMap;
-import java.util.List;
 
 public class Ship {
-    private int xCoord;
-    private int yCoord;
+    private int x;
+    private int y;
     private int direction;
     private int shipCounter;
     public HashMap<Integer, String> shipsType = new HashMap<Integer, String>();
-    public int shipSize;
+    public int size;
     private boolean completeShip;
 
 
-    public Ship(int xCoord, int yCoord, int direction, int shipSize) {
-        this.xCoord = xCoord;
-        this.yCoord = yCoord;
+    public Ship(int x, int y, int direction, int size) {
+        this.x = x;
+        this.y = y;
         this.direction = direction;
-        this.shipSize = shipSize;
+        this.size = size;
     }
 
+    public Ship(int x, int y) {
+        this(x, y, 0, 0);
+    }
 
-    public Ship(){}
 
+    public Ship() {
```

Comment:
Can a ship be created without any initialization of fields? If not, consider making this private